src/testing-library/queryByRef.ts

```diff
@@ -0,0 +1,32 @@
+import { queryHelpers } from '@testing-library/dom';
+import type { Matcher, MatcherOptions } from '@testing-library/dom/types/matches';
```

Comment:
Getting an eslint error here:
![image](https://user-images.githubusercontent.com/21334130/148749836-66d3ac91-960d-46e3-9b0c-0d534cf9fc7b.png)

We should add `/src/testing-library` to `.eslintrc.js`'s  `import/no-extraneous-dependencies` rule `devDependencies` directory exclude. As this directory will only be used inside of `*.test` files.